Bio/Align/__init__.py

```diff
@@ -1556,10 +1558,14 @@ def __setattr__(self, key, value):
     def align(self, seqA, seqB):
         """Return the alignments of two sequences using PairwiseAligner."""
         if isinstance(seqA, (Seq, MutableSeq)):
-            seqA = bytes(seqA)
+            sA = bytes(seqA)
+        else:
+            sA = seqA
         if isinstance(seqB, (Seq, MutableSeq)):
-            seqB = bytes(seqB)
-        score, paths = _aligners.PairwiseAligner.align(self, seqA, seqB)
+            sB = bytes(seqB)
+        else:
+            sB = seqB
+        score, paths = _aligners.PairwiseAligner.align(self, sA, sB)
```

Comment:
Could/should this catch plain (unicode) strings and turn them into bytes as well?

Reply:
`_aligners.PairwiseAligner.align` can handle `bytes` as well as `str`, including strings that include multi-byte characters. Those should be aligned as strings, not as bytes.